nservicebus/messaging/forwarding.md

```diff
@@ -9,13 +9,20 @@ related:
 
 Use this feature to forward successfully processed messages from an endpoint to a specific destination endpoint. Forwarding messages is particularly useful in complex upgrade scenarios when the old version and new version of a particular endpoint are running side-by-side.
 
+## Abort processing
+
+Forwarding should not be used to abort processing.  Messages cannot be forwarded to the `error` queue as no meta data would be present in the message meta data.  Forwarding to abort processing can result in partial updates.
+
+To abort processing throw an exception which ensures no ghost message are transmitted and any (transactional) state to be rolled back. 
+
+To prevent unnessary retries declare a [unrecoreable exception](/nservicebus/recoverability/#unrecoverable-exceptions) or create a [custom recoverability policy](/nservicebus/recoverability/custom-recoverability-policy.md).
```

Comment:
```suggestion
To prevent unnessary retries declare an [unrecoreable exception](/nservicebus/recoverability/#unrecoverable-exceptions) or create a [custom recoverability policy](/nservicebus/recoverability/custom-recoverability-policy.md).
```